src/Shared/Resources/Strings.shared.resx

```diff
@@ -48,107 +48,107 @@
   <resheader name="writer">
     <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.3500.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <data name="BuildAborted" UESanitized="false" Visibility="Public">
+  <data name="BuildAborted" Visibility="Public">
     <value>MSB4188: Build was canceled.</value>
     <comment>{StrBegin="MSB4188: "} Error when the build stops suddenly for some reason. For example, because a child node died.</comment>
   </data>
-  <data name="BuildEngineCallbacksInTaskHostUnsupported" UESanitized="false" Visibility="Public">
+  <data name="BuildEngineCallbacksInTaskHostUnsupported" Visibility="Public">
```

Comment:
Please don't remove `UESanitized` in this PR; it's not related to this work.

Reply:
True, but it doesn't hurt, and you previously recommended removing them all.